doc/source/whatsnew/v1.5.3.rst

```diff
@@ -32,7 +32,7 @@ Bug fixes
 
 Other
 ~~~~~
--
+- Added ``"truncate"`` option to ``if_exists`` argument in :meth:`DataFrame.to_sql` to truncate the existing table (:issue:`37210`)
```

Comment:
Can you move this to the 2.0 whatsnew?

Reply:
Done! Thanks for all your help with my first pandas PR!